pkg/u-boot/patches/patches-v2024.10-rc2/0004-Disable-mkeficapsule-tool.patch

```diff
@@ -0,0 +1,27 @@
+From 54888fa6e76563d3a66bea75c0b7095472eb3479 Mon Sep 17 00:00:00 2001
+From: Dimitrios Poulios <dpoulios85@gmail.com>
+Date: Mon, 4 Nov 2024 15:45:03 +0200
+Subject: [PATCH] Disable mkeficapsule tool
+
+Disable mkeficapsule tool to overcome riscv64 build error
```

Comment:
@dpoulios , this patch is not clear, what is causing the build issue with riscv64, the TOOLS_MKEFICAPSULE or the EFI_LOADER? If the former is the issue, you can just disable it from the default config:

```diff
diff --git a/configs/qemu-riscv64_smode_defconfig b/configs/qemu-riscv64_smode_defconfig
index 6baad1de38..bd58f4cc90 100644
--- a/configs/qemu-riscv64_smode_defconfig
+++ b/configs/qemu-riscv64_smode_defconfig
@@ -23,3 +23,4 @@ CONFIG_SYS_RELOC_GD_ENV_ADDR=y
 CONFIG_DM_MTD=y
 CONFIG_FLASH_SHOW_PROGRESS=0
 CONFIG_SYS_MAX_FLASH_BANKS=2
+# CONFIG_TOOLS_MKEFICAPSULE is not set
```

Reply:
But I think disable only for riscv64 is enough, right? I mean, you just need to use the patch above and let the TOOLS_MKEFICAPSULE enabled for other architectures....

Reply:
There are 2 solutions:
- either remove the gnutls-dev package (and util-linux-dev, too, which is a similar case) from BUILD_PKGS and disable the mkeficapsule tool (which is the one that needs these packages) in all architectures
- or, add the packages in BUILD_PKGS_amd64 & BUILD_PKGS_arm64 (as they are not available for risc) and disable the tool only in risc

I did the first one. The mkeficapsule tool is not needed for our build